src/Core/Components/Nav/FluentNavItem.razor.cs

```diff
@@ -0,0 +1,392 @@
+// ------------------------------------------------------------------------
+// This file is licensed to you under the MIT License.
+// ------------------------------------------------------------------------
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.AspNetCore.Components.Routing;
+using Microsoft.AspNetCore.Components.Web;
+using Microsoft.FluentUI.AspNetCore.Components.Utilities;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+/// <summary>
+/// Represents a navigation menu item that renders content within a Fluent UI styled navigation link.
+/// </summary>
+public partial class FluentNavItem : FluentComponentBase, IAsyncDisposable
```

Comment:
Since you inherit from `FluentComponentBase` you don't need to specify `IAsyncDisposable` here again

Reply:
Done